tests/unit/plugins/module_utils/test_vm.py

```diff
@@ -598,9 +605,10 @@ def test_post_vm_payload_cloud_init_absent(self, rest_client):
             boot_devices=[],
             attach_guest_tools_iso=False,
             operating_system=None,
+            machine_type="BIOS",
         )
 
-        post_vm_payload = vm.post_vm_payload(rest_client, {})
+        post_vm_payload = vm.post_vm_payload(rest_client, {"machine_type": "BIOS"})
```

Comment:
This change looks wrong, `{"machine_type": "BIOS"}` would land in cloud_init data - that does not make sense.

Reply:
I'm not sure, this was Tjazs change. But I think he is unit testing post payload without cloud_init. And machine types is apart of the post body without cloud_init?

Reply:
The line `post_vm_payload = vm.post_vm_payload(rest_client, {"machine_type": "BIOS"})` is only part of unit test. I reverted it, and unit tests still pass, so I commit this.